### PR TITLE
feat: `IDTokenClaims` type

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

1. add `IDTokenClaims` type
2. add PR template

since API response types will come with API functions, the only type we need to define in this PR is ID Token Claims.

## Reviewer

@logto-io/eng

### CODE REVIEW

File: Sources/Logto/Types/IdTokenClaims.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+//  IdTokenClaims.swift
+//  
+//
+//  Created by Gao Sun on 2022/1/7.
+//
+
+import Foundation
+
+public struct IDTokenClaims {
```
Comment (IceHe) [inline Sources/Logto/Types/IdTokenClaims.swift:10]: `IdTokenClaims`: `d` should be in lower case according to the [documentation](https://www.notion.so/silverhand/9c1fbcd630fc44adaa45a6d7d686326a?v=3cea92e2906048d287a8648b76c73c65)

Comment (gao-sun) [reply Sources/Logto/Types/IdTokenClaims.swift:10]: good call

**Review by IceHe — APPROVED** [review]
LGTM

File: Sources/Logto/Types/IdTokenClaims.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import Foundation
+
+public struct IdTokenClaims {
+    let sub: String
+    let atHash: String
+    let aud: String
+    let exp: Int64
+    let iat: Int64
+    let iss: String
```
Comment (Hengyu) [inline Sources/Logto/Types/IdTokenClaims.swift:16]: These properties are `internal` by default, therefore user cannot get properties of `IdTokenClaims` outside the module. E.g., `print(tokerClaims.sub) // error, the property is internal`.

Not sure if this is by design, if we don't want to expose those properties, just leave this comment :)

**Pull request merged**